app/models/plant_population.rb

```diff
@@ -25,6 +25,7 @@ class PlantPopulation < ActiveRecord::Base
 
   validates :name,
             presence: true,
+            uniqueness: true,
             allow_blank: true
```

Comment:
It would allow any number of NULL records - however you are right in that it doesn't make much sense. I've checked and this field is not blank in any existing PP records; accordingly, I will remove this clause.